### PR TITLE
[wmcb] Add Makefile target for unit test binary

### PR DESCRIPTION
**Pull request by aravindhp**
- Add a Makefile target called "build-wmcb-unit-test", that will build a Windows unit test binary
- Update the README with instructions on how to run unit tests

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -48,6 +48,19 @@ Repeat the above steps for the `system:node:$NODE_NAME` csr which will appear sh
 
 You can now do `oc get nodes`, and see the Windows node has joined the cluster.
 
+#### Unit testing
```
Comment (sebsoto) [inline docs/README.md]: If we are keeping the `test-unit` target, then you might want to add that if you are developing on windows you should run `make test-unit` to run the tests.

Comment (sebsoto) [reply docs/README.md]: Either that or we can remove the target

File: Makefile [hunk 2]
```diff
@@ -14,6 +14,9 @@ GO_BUILD_ARGS=CGO_ENABLED=0 GO111MODULE=on
 build:
 	$(GO_BUILD_ARGS) GOOS=windows go build -o wmcb.exe  $(MAIN_PACKAGE)
 
+build-wmcb-unit-test:
+	GOOS=windows GOFLAGS=-v go test -c ./pkg/... -o wmcb_unit_test.exe
+
```
[the remark this answers is not in the document]
Comment (sebsoto) [reply Makefile]: i'm +1 on removing it

**Comment by aravindhp** [comment]
@sebsoto @ravisantoshgudimetla I have made the changes you requested. PTAL.

**Comment by sebsoto** [comment]
/hold 
so @ravisantoshgudimetla can take a look

**Comment by ravisantoshgudimetla** [comment]
/test test-e2e-tools 

Infra flake

**Pull request merged**